frontend/src/assets/css/normalize.css

```diff
@@ -87,9 +94,12 @@ a {
  */
 
 abbr[title] {
-	border-bottom: none; /* 1 */
-	text-decoration: underline; /* 2 */
-	text-decoration: underline dotted; /* 2 */
+	border-bottom: none;
+	/* 1 */
```

Comment:
¿Por qué has modificado el normalize.css?

Reply:
para cargar la fuente que necesitaba en el span del logo
@font-face {
	font-family: "Roboto Mono Medium for Powerline";
	src: local("Roboto Mono Medium for Powerline"), url("../fonts/Roboto Mono Medium for Powerline/Roboto Mono Medium for Powerline.ttf") format("truetype");
}
esto es lo único que toqué